### ISSUE TITLE
Flaky PathTest.testHugeAppend

### ISSUE THREAD

**Issue by puzpuzpuz** [questdb/questdb#4098]
### To reproduce

Occurred on arm64 CI worker:
```
2024-01-03T09:50:50.4471479Z 2024-01-03T09:50:50.446156Z I i.q.t.TestListener >>>> io.questdb.test.std.str.PathTest.testHugeAppend
2024-01-03T09:50:50.4673345Z 2024-01-03T09:50:50.452842Z E i.q.t.TestListener ***** Test Failed *****io.questdb.test.std.str.PathTest.testHugeAppend duration_ms=6 : 
2024-01-03T09:50:50.4674574Z java.lang.OutOfMemoryError
2024-01-03T09:50:50.4675259Z 	at jdk.internal.misc.Unsafe.allocateMemory(Unsafe.java:616)
2024-01-03T09:50:50.4675997Z 	at sun.misc.Unsafe.allocateMemory(Unsafe.java:462)
2024-01-03T09:50:50.4676651Z 	at io.questdb.std.Unsafe.malloc(Unsafe.java:72)
2024-01-03T09:50:50.4677274Z 	at io.questdb.std.Unsafe.calloc(Unsafe.java:139)
2024-01-03T09:50:50.4678029Z 	at io.questdb.test.std.str.PathTest.testHugeAppend(PathTest.java:161)
2024-01-03T09:50:50.4678836Z 	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2024-01-03T09:50:50.4679694Z 	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2024-01-03T09:50:50.4680669Z 	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2024-01-03T09:50:50.4681560Z 	at java.lang.reflect.Method.invoke(Method.java:566)
2024-01-03T09:50:50.4682422Z 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
2024-01-03T09:50:50.4683378Z 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
2024-01-03T09:50:50.4684323Z 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
2024-01-03T09:50:50.4685253Z 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
2024-01-03T09:50:50.4686183Z 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
2024-01-03T09:50:50.4687094Z 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
2024-01-03T09:50:50.4687966Z 	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
2024-01-03T09:50:50.4688818Z 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
2024-01-03T09:50:50.4689709Z 	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
2024-01-03T09:50:50.4690791Z 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
2024-01-03T09:50:50.4691662Z 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
2024-01-03T09:50:50.4692697Z 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
2024-01-03T09:50:50.4693558Z 	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
2024-01-03T09:50:50.4694389Z 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
2024-01-03T09:50:50.4695209Z 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
2024-01-03T09:50:50.4696049Z 	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
2024-01-03T09:50:50.4696880Z 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
2024-01-03T09:50:50.4697723Z 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
2024-01-03T09:50:50.4698750Z 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
2024-01-03T09:50:50.4699610Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:264)
2024-01-03T09:50:50.4700542Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:153)
2024-01-03T09:50:50.4701481Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:124)
2024-01-03T09:50:50.4702448Z 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
2024-01-03T09:50:50.4703535Z 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
2024-01-03T09:50:50.4704444Z 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
2024-01-03T09:50:50.4705820Z 2024-01-03T09:50:50.452966Z I i.q.t.TestListener <<<< io.questdb.test.std.str.PathTest.testHugeAppend duration_ms=6
```

### QuestDB version:

latest master

### OS, in case of Docker specify Docker and the Host OS:

Amazon Linux

### File System, in case of Docker specify Host File System:

xfs

### Full Name:

Andrei Pechkurov

### Affiliation:

QuestDB

### Have you followed Linux, MacOs kernel configuration steps to increase Maximum open files and Maximum virtual memory areas limit?

- [X] Yes, I have

**Comment by bziobrowski** [comment]
This test is fixed in https://github.com/questdb/questdb/pull/4066 .

**Comment by puzpuzpuz** [comment]
> This test is fixed in #4066 .

Great. Updated PR description to mark it as resolving this issue.
